lib/validateWebpackOptions.js

```diff
@@ -9,11 +9,11 @@ var ajv = new Ajv({
 	allErrors: true,
 	verbose: true
 });
-var validate = ajv.compile(webpackOptionsSchema);
 
-function validateWebpackOptions(options) {
+function validateWebpackOptions(options, customSchema) {
+	var schema = customSchema || webpackOptionsSchema;
```

Comment:
better remove the default and always pass the schema. remove the require. pass the schema from caller side. rename the function and file to `validateOptions`.